generator/template/model_template.go

```diff
@@ -268,6 +269,11 @@ func getGoType(column metadata.Column) interface{} {
 
 // toGoType returns model type for column info.
 func toGoType(column metadata.Column) interface{} {
+	// We don't support multi-dimensional arrays
+	if column.DataType.Dimensions > 1 {
+		return ""
+	}
+
```

Comment:
Multi dimensional arrays are just arrays containing other arrays. In our  case `Array[Array[StringExpression]]`. But it is fine with PR to go with single dimension arrays only.

Reply:
You're right, but since we don't have a good model type yet, I haven't included this case yet